applications/crossbar/src/modules/cb_websockets.erl

```diff
@@ -17,13 +17,32 @@
         ]).
 
 -include("../crossbar.hrl").
+-include_lib("whistle/include/wapi_conf.hrl").
 
 -define(CB_LIST, <<"websockets/crossbar_listing">>).
+
+
+-define(CALL, [<<"call.CHANNEL_CREATE.*">>
+               ,<<"call.CHANNEL_ANSWER.*">>
+               ,<<"call.CHANNEL_DESTROY.*">>
+               ,<<"call.CHANNEL_BRIDGE.*">>
+              ]).
+
+-define(
+    OBJECTS
+    ,lists:merge(
```

Comment:
No need to have nested list comprehensions + a call to `lists:merge/1`. This can be accomplished with one list comprehension. Please make that change.